bounties_api/user/views.py

```diff
@@ -117,6 +117,16 @@ def get(self, request, public_address):
         fulfiller_fulfillment_acceptance = None if not user_fulfillments.count() else (
             user_fulfillments.filter(accepted=True).count() / user_fulfillments.count())
 
+        total_fulfillments_on_bounties = user_bounties.annotate(
```

Comment:
@c-o-l-o-r @codeluggage  pretty sure there's a WAY easier way to do this:
`user_bounties.fulfillments.count()`
Does that not work?

Reply:
I tried this a few other things, but was unable to get it to work. I feel like there should be something easier, but I could not figure it out 😢.